spec.md

```diff
@@ -232,12 +233,14 @@ Policies MUST specify the following fields:
   for this domain might be handled by any MX whose hostname is a subdomain of
   "example.com" or "example.net."
 * a: The mechanism to use to authenticate this policy itself. See the section
-  _Policy_ _Authentication_ for more details. Possible values are:
-  * webpki:URI, where URI points to an HTTPS resource at the recipient domain
-    that serves the same policy text.
+  _Policy_ _Discovery_ _&_ _Authentication_ for more details. Possible values are:
+  * webpki: Indicating that the policy is expected to be served over HTTPS on a
+    well-known URI: https://policy._smtp_sts.example.com/current. The 
+    example.com is the recipient's domain, and the endpoint serves the same policy
+    text.
   * dnssec: Indicating that the policy is expected to be served over DNSSEC.
 * c: Constraints on the recipient MX's TLS certificate (plain-text, required).
-  See the section _Policy_ _Validation_ for more details. Possible values are:
+  See the section _Recipient's_ _Certificate_ _Validation_ for more details. Possible values are:
```

Comment:
Policy validation is fine, but what we described in that section is only part of the policy validation. Like i mentioned before, cert validation is only one part. The other part is the MX name check. Based on the description, I proposed a heading change to make it more explicit. If you would like to keep 'Policy Validation', make sure we describe all validation steps in that section.